### ISSUE TITLE
Display a title tooltip on mouse-hover/tab of the sort icon in Data Table

### ISSUE THREAD

**Issue by johnharan** [carbon-design-system/carbon#1411]
In chrome v70, if the user hovers over a sort icon in the Table component, the title e.g. "Sort rows by this header in ascending order" will show in a tooltip.

![sort_title_tooltip](https://user-images.githubusercontent.com/8806230/48192744-8d219300-e340-11e8-9f4b-6bd4216d8bfb.png)

The title does not show in firefox 63, safari v12, IE 11.

For enhanced accessibility, the tootip should show on mouse hover or on keyboard tab.

> Is this a feature request (new component, new icon), a bug, or a general issue?

This is a feature/enhancement request.

> Is this issue related to a specific component?

Table Component

> What version of the Carbon Design System are you using?

"carbon-components": "9.32.1",
"carbon-components-angular": "1.9.1"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

## Steps to reproduce the issue

https://angular.carbondesignsystem.com/?knob-size=md&knob-showSelectionColumn=true&knob-striped=true&knob-sortable=true&selectedKind=Table&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

1.  hover over the sort icon in the table component in chrome and note that the tooltip will display the title contained in the sort svg.
2.  Repeat step 1 on firefox/IE/safari and note that no tooltip displays

**Comment by johnharan** [comment]
Adding the title attribute to the button which wraps the sort icon seems to have the desired result of the tooltip showing in Firefox.. and i think it makes sense to add the title to the button, because the user can click anywhere within the button to trigger the sort, so hovering anywhere within the button should show the title.

So for example the html could be:

```html
<button title="Sort rows by this header in descending order" class="bx--table-sort-v2 bx--table-sort-v2--active" data-event="sort" data-previous-value="descending">
      <span class="bx--table-header-label">Protocol</span>
      <svg class="bx--table-sort-v2__icon" width="10" height="5" viewBox="0 0 10 5" aria-label="Sort  rows by this header in descending order">
             <title>Sort rows by this header in descending order</title>
             <path d="M0 0l5 4.998L10 0z" fill-rule="evenodd"></path>
       </svg>
</button>
```

Note the **title="Sort rows by this header in descending order"** added to the button.

Also the **alt** attribute should be removed from the svg.

**Comment by asudoh** [comment]
Ping @jnm2377 @vpicone to see if there is an interest in tackling this before being closed.

**Comment by dakahn** [comment]
The title attribute is a pretty unreliable way of communicating to your users and I don't think it's the solution we would want here (though a 🤷‍♂ better than nothin' argument could be made).

https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/

In short whatever you put in `title` is hidden from users on many devices and from users with certain motor/cognitive impairments.

**Comment by GabrieleBarboni** [comment]
Still needed

**Comment by JoannaCai** [comment]
Problem exists in sorted Datatable on FF 60.8.1 esr only,  could someone have a look at?
